channels/google-actions/src/main/kotlin/com/justai/jaicf/channel/googleactions/jaicp/ActionsFulfillmentDialogflow.kt

```diff
@@ -9,4 +9,9 @@ class ActionsFulfillmentDialogflow(
 ) : JaicpCompatibleChannelFactory {
     override val channelType = "dialogflow"
     override fun create(botApi: BotApi) = ActionsFulfillment.dialogflow(botApi, useDataStorage)
+
+    companion object : JaicpCompatibleChannelFactory {
```

Comment:
Let's use [delegation](https://kotlinlang.org/docs/delegation.html) for easier inheritance
```suggestion
    companion object : JaicpCompatibleChannelFactory by ActionsFulfillmentDialogflow()
```